src_files.yml

```diff
@@ -14,7 +14,7 @@ pulp_cluster:
     rtl/core_demux.sv,
     rtl/cluster_interconnect_wrap.sv,
     rtl/tcdm_banks_wrap.sv,
-    rtl/periph_demux.sv,
+    # rtl/periph_demux.sv,
```

Comment:
It's up to @OttG to decide, but you may want to remove these files entirely when they are removed from the file list of IPApproX and Bender. I find it confusing when source code files are around that are not actually used (returns misleading `grep` results, etc).

Reply:
+1

Reply:
I completely agree this is confusing, which is why I started with removing them from the build. I have removed the files, as well as the lines from `Bender.yml` and `src_files.yml`. @OttG Please let me know if you would like me to revert this change.